samples/examples/income_classifier/train.md

```diff
@@ -169,98 +169,89 @@ train_pipeline = Pipeline(steps=[('preprocessor',preprocessor),('classifier',clf
-
-
+```
```

Comment:
:information_source: Unfortunately this is another false positive, this time due to suitable indentation to be detected as an output block.

I don't think this is terrible because:
1. The above block of HTML already renders poorly and is not human-readable
2. [This](https://github.com/SeldonIO/seldon-core/blob/10a8dff21bf04fabe326a258197cb02470326576/samples/examples/income_classifier/train.md) renders in practically the same way as [what's already there](https://github.com/SeldonIO/seldon-core/blob/fb442883d7e9014405f5675f79f7164cf2619359/samples/examples/income_classifier/train.md), presumably due to the divs?

I think the best solution here is actually just to remove the offending content/modify the generating cell appropriately, as it's very much an edge case.